index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Static Web HTML + CSS Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<body>
+<!-- Yellow post it section -->
+  <article class="card">
+    <header>
+      <h1>Yellow Post-it <br> Notes</h1>
+    </header>
+      <section>
+        <img src="http://www.clipartbest.com/cliparts/dc8/deo/dc8deozce.png" alt="Yellow Post it Note" class="pics">
```

Comment:
So this totally works and is fine. But technically the requirement asks that 'The first section should contain three child block elements.' and 5-i asks 'The first element contains the product image.'
It would be great practice to put your images in a block level element and then try to recenter them